projects/elife.yaml

```diff
@@ -363,3 +371,48 @@ search:
         ports:
             - 22
             - 80
+
+bus:
+    default-branch: null
+    domain: null
+    int_domain: null
+    aws:
+        ec2: false
+        sns:
+            # bus-articles-prod, bus-articles-end2end, etc.
+            - bus-articles--{cluster}
```

Comment:
ok, so a stackname looks like:
<project>--<instanceid>[--clusterid]

so: lax--prod, lax--end2end, lax--2015-01-01, lax--foo

and in a load balanced setup:
lax--prod--1, lax--prod--2, lax--prod--3

would you have:
bus-articles--1 or bus-articles--end2end ?